app/controllers/public/sessions.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { orderBy, groupBy } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  sessions: [{ shortAbstract: 'An introduction to the event', id: '1', title: 'Welcome to FOSSASIA', startAt: '10:35 AM / 17-03-2017', track: { color: 'green', fontColor: 'green', id: 0, name: 'Track 1' }, microlocation: { id: 0, name: 'Room 1' }, speakers: [{ shortBiography: 'Works for ORG 1', id: 0, city: 'Delhi', name: 'Arnold Singh', speakingExperience: 'GSOC 2015', organisation: 'ORG 1', longBiography: '', photo: { 'iconImageUrl': '' } }] },
```

Comment:
Like for example, 
- in this page we may want to display it in a more human readable form like `March 23, 2017 05:30 PM`
- In the sessions table we'll display `23/03/2017 05:30 PM`
- In the form we may display in another way for it to work with the date picker.

We can't and shouldn't put all of this inside the model.